dbms/src/Storages/DeltaMerge/StableValueSpace.cpp

```diff
@@ -564,5 +564,53 @@ StableValueSpace::Snapshot::getAtLeastRowsAndBytes(const DMContext & context, co
     return ret;
 }
 
+static size_t defaultValueBytes(const Field & f)
+{
+    switch (f.getType())
+    {
+    case Field::Types::Decimal32:
+        return 4;
+    case Field::Types::UInt64:
+    case Field::Types::Int64:
+    case Field::Types::Float64:
+    case Field::Types::Decimal64:
+        return 8;
+    case Field::Types::UInt128:
+    case Field::Types::Int128:
+    case Field::Types::Decimal128:
+        return 16;
+    case Field::Types::Int256:
+    case Field::Types::Decimal256:
+        return 32;
+    case Field::Types::String:
+        return f.get<String>().size();
```

Comment:
Usually default value of string is `""` or `NULL`, which may cause inaccuracy.

Can we get `255` for type `varchar(255)`?

Reply:
If the type is `varchar(255)`, default `""`, but the length of the data in this col is larger than `200`?